dash-info.yaml

```diff
@@ -1,10 +1,2612 @@
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlOl(list(
+                  htmlLi("Montreal"),
+                  htmlLi("Toronto"),
+                  htmlLi("Halifax")
+                )),
+                htmlUl(list(
+                  htmlLi("Montreal"),
+                  htmlLi("Toronto"),
+                  htmlLi("Halifax")
+                ))
+              ))
+            )
+
+            app$run_server()
+    - name: htmlLink
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlLink(rel = "stylesheet", type = "text/css", href = "https://codepen.io/chriddyp/pen/bWLwgP.css")
+              ))
+            )
+
+            app$run_server()
+    - name: htmlListing
+      dontrun: TRUE
+      code: |
+            # Warning: The <listing> element was intended as a way to render HTML code on a page.
+            # It was never properly supported, and is now deprecated. Using <listing> will almost
+            # certainly result in unexpected results. Instead, use <code>, or place the content in
+            # a <div> with the appropriate CSS styling.
+
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlListing(list(
+                  htmlUl("A"),
+                  htmlUl("B"),
+                  htmlUl("C")
+                ))
+              ))
+            )
+
+            app$run_server()
+    - name: htmlMain
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlMain(
+                  list(htmlH1("Benjamin Franklin"),
+                       htmlP("Benjamin Franklin was an American polymath
+                             and one of the Founding Fathers of the United States.
+                             Franklin was a leading author, printer, political theorist,
+                             politician, Freemason, postmaster, scientist, inventor,
+                             humorist, civic activist, statesman, and diplomat."))
+                )
+              ))
+            )
+
+            app$run_server()
+    - name: htmlMapEl
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlImg(src = "https://upload.wikimedia.org/wikipedia/commons/0/0c/PIA17351-ApparentSizes-MarsDeimosPhobos-EarthMoon.jpg",
+                        useMap = "#image-map"),
+                htmlMapEl(list(
```

Comment:
is this the only one that has a different name from its HTML counterpart (`<map>`)? we should add a comment about this (and to any others if there are any) so users can find regular HTML documentation about it.

Reply:
fixed in 5caaeda